imap_processing/swe/l1a/swe_science.py

```diff
@@ -133,7 +133,8 @@ def swe_science(decom_data: list, data_version: str) -> xr.Dataset:
         # where 1260 = 180 x 7 CEMs
         # Take the "raw_counts" indices/counts mapping from
         # decompression_table and then reshape the return
-        uncompress_data = np.take(decompression_table, raw_counts).reshape(180, 7)
+        place_holder: np.ndarray = np.take(decompression_table, raw_counts)
+        uncompress_data = place_holder.reshape(180, 7)
```

Comment:
Rather than the placeholder variables to appease the type checkers, I think I'd prefer to just ignore them. Not sure how you and others feel about that though, but I personally find this harder to read than just ignoring the type checking error.

`# type: ignore[attr-defined]`